src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeStatisticsTableViewCell.swift

```diff
@@ -86,6 +86,9 @@ class HomeStatisticsTableViewCell: UITableViewCell {
 	) {
 		guard cellModel == nil else { return }
```

Comment:
What is about cell reusing? Cells can be reused and configured for another indexpath. Then this configure will be called with data for the corresponding (new) indexpath. But it won't be reconfigured? Is cell reusing somehow disabled?